src/module.ts

```diff
@@ -25,17 +31,18 @@ export default defineNuxtModule<ModuleOptions>({
     name: 'nuxt-directus',
     configKey: 'directus',
     compatibility: {
-      nuxt: '^3.0.0-rc.9 || ^2.16.0',
-      bridge: true
+      nuxt: '^3.0.0'
     }
   },
   defaults: {
-    url: process.env.NUXT_DIRECTUS_URL
+    url: 'http://localhost:8055' as string,
+    staticToken: '' as string,
```

Comment:
Please dont use type casts

Reply:
Would you mind to explain why? I'm still learning js/ts and I'm curious to understand what this could cause

Reply:
You should always be careful with type casts, in principle a type cast makes sense if you know more about the type than the TypeScript compiler. 

With a type cast you force the type, you only use them to satisfy the TypeScript error. And that is not right.
Type casting is particularly useful when you're working with dynamic data.

In this case its a string, and this is clear. So dont cast it.